src/canister_tests/src/api/internet_identity.rs

```diff
@@ -16,7 +16,7 @@ pub fn health_check(env: &StateMachine, canister_id: CanisterId) {
     let user_number: types::AnchorNumber = 0;
     // XXX: we use "IDLValue" because we're just checking that the canister is sending
     // valid data, but we don't care about the actual data.
-    let _: (candid::parser::value::IDLValue,) =
+    let _: (candid::types::value::IDLValue,) =
```

Comment:
Was the package moved or are you using something else now?

Reply:
It was moved, see [here](https://github.com/dfinity/candid/blob/master/Changelog.md#breaking-changes).